src/sentry/notifications/notifications/activity/base.py

```diff
@@ -129,12 +129,32 @@ def get_context(self, provider: ExternalProviders | None = None) -> MutableMappi
         should_add_url = provider is not None
         text_description = self.description_as_text(text_template, params, should_add_url, provider)
         html_description = self.description_as_html(html_template or text_template, params)
-        return {
+
+        context = {
             **self.get_base_context(),
             "text_description": text_description,
             "html_description": html_description,
         }
 
+        # Add suspect commits to workflow notifications using the same method as issue alerts
+        if self.group:
+            # Create a minimal event object since get_commits() only uses event.group_id
+            # Include empty data/platform to avoid AttributeError if fallback analysis is triggered
+            class MinimalEvent:
+                def __init__(self, group_id):
+                    self.group_id = group_id
+                    self.data = {}  # Empty data to avoid AttributeError
+                    self.platform = None  # Platform info not needed
+
+            try:
+                minimal_event = MinimalEvent(self.group.id)
+                context["commits"] = get_commits(self.project, minimal_event)
```

Comment:
it seems like this part of the function would use other things on the event, right? https://github.com/getsentry/sentry/blob/56ef263316fa056a05094f8467b0297e246100bb/src/sentry/utils/committers.py#L346

maybe we should make a separate method which takes in only group id and returns the serialized suspect commit?